src/Elsa.CustomActivities/Activities/Common/CheckboxModel.cs

```diff
@@ -5,5 +5,5 @@ public class CheckboxModel
         public ICollection<CheckboxRecord> Choices { get; set; } = new List<CheckboxRecord>();
     }
 
-    public record CheckboxRecord(string Identifier, string Answer, bool IsSingle, bool IsPrePopulated);
+    public record CheckboxRecord(string Identifier, string Answer, bool IsSingle, bool IsPrePopulated, bool IsExclusiveToQuestion);
```

Comment:
We do not need this on a standard CheckboxRecord, as it is only relevant for WeightedCheckboxRecord

Reply:
This is now removed.